packages/cli/generators/project/templates/README.md.ejs

```diff
@@ -1,3 +1,15 @@
 # <%= project.name %>
 
 [![LoopBack](https://github.com/strongloop/loopback-next/raw/master/docs/site/imgs/branding/Powered-by-LoopBack-Badge-(blue)-@2x.png)](http://loopback.io/)
+
+## Installation
+
+Describe the installation steps.
+
+## Basic Use
+
+Describe the usage.
+
+## License
+
+Specify the license.
```

Comment:
I suspect that many people won't bother to edit this section and push it to github & npmjs as-is. I think it's better to leave out the License section completely, rather than have content that's meaningless.